src/components/Calender.vue

```diff
@@ -60,31 +71,29 @@
 	const firstDay = firstDayInMonthIndex(props.year, props.month)
 
 	const paddingCalenderItems = Array(firstDay).fill("").map<CalenderItem>((_val, index) => {
-		return {isPadding: true, date: index - firstDay}
+		return { isPadding: true, date: index - firstDay }
 	})
-	
+
 	const calenderDays = Array(numberOfDays).fill("").map<CalenderItem>((_val, index) => {
-		return {isPadding: false, date: index}
+		return { isPadding: false, date: index }
 	})
-	
+
 	const calender = [...paddingCalenderItems, ...calenderDays]
 </script>
 
 <style scoped>
-	#calender
-	{
+	#calender {
```

Comment:
I am also against placing the brace on the same line in CSS (I will concede when it comes to JS/TS for other reasons, but they are different languages that don't need to be formatted the same way). If you want to change the code style here, I would split this off into its own PR, and we can discus.

Reply:
Ya same problem here I'll try to figure out how to turn this off